helper/communicator/config.go

```diff
@@ -66,6 +68,23 @@ type Config struct {
 	WinRMTransportDecorator func() winrm.Transporter
 }
 
+// ReadSSHPrivateKeyFile returns the SSH private key bytes
+func (c *Config) ReadSSHPrivateKeyFile() ([]byte, error) {
```

Comment:
Nice !! 👍

Reply:
Do you think adding a `ReadSSHBastionPrivateKeyFile` func too would be a good idea ?

Reply:
As I see, `SSHBastionPrivateKeyFile` is using only in `step_connect_ssh.go` in communicator, so it's not necessary. I'll think about adding `homedir.Expand` to `helperssh.FileSigner` before file open.

Reply:
ok 🙂